DockerBuild.ps1

```diff
@@ -0,0 +1 @@
+docker build . --isolation process -t whales-names
```

Comment:
remove process iso as it is not necessary

Reply:
I left this in so that ppl don't need hyper v, al they need is to update to windows 1809 which is probably already done